src/radiate.gleam

```diff
@@ -18,24 +18,37 @@ pub type NoInitializer
 /// Phantom type to indicate builder has initializer
 pub type HasInitializer
 
+/// Phantom type to indicate builder has no callback
+pub type NoCallback
+
+/// Phantom type to indicate builder has callback
+pub type HasCallback
```

Comment:
Is this really necessary? It's set, sure, but it isn't really used for validation.

Reply:
What do you mean? It's used in `on_reload`.

Reply:
Oh I missed that, thank you! Makes sense. Merging then, thank you!